.github/workflows/build.yml

```diff
@@ -1,19 +1,23 @@
 name: Deploy spec
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
 
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - run: npm install
-    - run: npm run build
-    - name: commit changes
-      uses: elstudio/actions-js-build/commit@v3
-      with:
-        commitMessage: "fixup: [spec] `npm run build`"
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js (LTS)
+        uses: actions/setup-node@v2-beta
+      - name: Install
+        run: npm install
+      - name: Build
+        run: npm run build
```

Comment:
i prefer the name of the step to be the command itself, rather than a prose string that requires me looking to see what comand it runs.